### PR TITLE
TM-3731: Client Profiles - Combine Location & Org

### PR DESCRIPTION
**Pull request by pveland621**
Added organization field (pos_org_short_desc) to userProfile payload to display on Client Profile cards/rows. Ticket was originally asking for the long description of organizations as well but after DEV1 investigation, it was determined that different data from a separate table is needed to expose the correct long descriptions, which will be addressed in a future ticket

Dual Merge: 
- [Mock PR](https://github.com/MetaPhase-Consulting/mock-fsbid/pull/326)

### CODE REVIEW

File: src/Components/BidderPortfolio/BidderPortfolioStatCard/BidderPortfolioStatCard.jsx [hunk 1]
```diff
@@ -17,6 +17,10 @@ const BidderPortfolioStatCard = ({ userProfile, classifications }) => {
   const ted = formatDate(get(userProfile, 'current_assignment.end_date'));
   const languages = get(userProfile, 'current_assignment.position.language');
   const bidder = get(userProfile, 'shortened_name') || 'None listed';
+
+  const orgShort = get(userProfile, 'current_assignment.position.organization');
+  const orgLong = get(userProfile, 'current_assignment.position.organization_long');
```
Comment (scott062) [inline src/Components/BidderPortfolio/BidderPortfolioStatCard/BidderPortfolioStatCard.jsx:22]: Remove orgLong and related formatting for it

File: src/sass/_bidderPortfolio.scss [hunk 2]
```diff
@@ -269,6 +269,10 @@
     font-weight: bold;
   }
 
+  .location-label {
+    flex-shrink: 0
```
Comment (scott062) [inline src/sass/_bidderPortfolio.scss:273]: ;

**Review by scott062 — APPROVED** [review]
Just add the `;` to styling, otherwise looks good

File: src/sass/_bidderPortfolio.scss (hunk 2 above)
Comment (pveland621) [reply src/sass/_bidderPortfolio.scss]: ;

**Pull request merged**